docs/FeatureManagement/Clients/SpringCloud/Spring.FeatureFlag.v1.0.0.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://azconfig.io/schemas/FeatureManagement/v1.0.0/SpringFeatureFlag.json",
+    "type": "object",
+    "title": "An Spring Cloud Azure App Configuration Feature Declaration",
+    "required": [
+        "feature-management"
+    ],
+    "properties": {
+        "feature-management": {
+            "$id": "#/properties/feature-management",
+            "type": "object",
+            "title": "The Feature Management Schema",
+            "properties": {
+                "patternProperties": {
+                    ".*": {
+                        "anyOf": [
+                            {
+                                "type": "boolean",
+                                "title": "On/Off Feature Flag",
+                                "description": "A feature flag that always returns the same value.",
+                                "examples": [
+                                    true,
+                                    false
+                                ]
+                            },
+                            {
+                                "type": "object",
+                                "title": "Conditional Feature Flag",
+                                "description": "A feature flag which value is dynamic based on a set of feature filters",
+                                "default": null,
+                                "properties": {
+                                    "enabled-for": {
+                                        "$id": "#/properties/feature-management/properties/.*/properties/EnabledFor",
+                                        "type": "array",
+                                        "title": "The declaration of conditions used to dynamically enable features.",
```

Comment:
Is it the declaration of conditions, or is it a list of feature filters which should be used to evaluate the state of the feature?

Reply:
I would've expected something like: "A list of feature filters that can be used to enable the feature conditionally."